src/common/tabs/FocalboardTabView.ts

```diff
@@ -1,12 +1,14 @@
 // Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {getFormattedPathName} from 'common/utils/url';
+
 import BaseTabView from './BaseTabView';
 import {TabType, TAB_FOCALBOARD} from './TabView';
 
 export default class FocalboardTabView extends BaseTabView {
     get url(): URL {
-        return this.server.url;
+        return new URL(`${this.server.url.origin}${getFormattedPathName(this.server.url.pathname)}plugins/focalboard`);
```

Comment:
this might change in the future, right? should we create a function that abstracts it? 0/5 though

Reply:
This would be the only source of truth for the app. I figure if we ever allow for it to be modified per server, then maybe we'd have to add it to a configuration or something. But for now I think it works here.